ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -33,19 +34,26 @@ spec:
     - name: github_usernames
       description: List of GitHub accounts with permissions allowing operator submission
   workspaces:
-    - name: pyxis-ssl-credentials
-      optional: true
     - name: source
     - name: results
+  volumes:
+    - name: pyxis-ssl-volume
+      secret:
+        secretName: "$(params.pyxis_ssl_secret_name)"
+        optional: true
   steps:
     - name: get-cert-project-related-data
       image: "$(params.pipeline_image)"
       workingDir: $(workspaces.source.path)
       env:
         - name: PYXIS_CERT_PATH
-          value: $(params.pyxis_cert_path)
+          value: /etc/pyxis-ssl-volume/$(params.pyxis_ssl_cert_secret_key)
```

Comment:
Python scripts that talk to Pyxis use API key or Cert auth based on if a cert path is set. In the old code, the default value for cert path was set to `""` which caused the script used an API key. Now when you always set the cert path how does the script know when to use a cert and when to use API key?

Reply:
This was fixed in the latest pipeline image with a check for valid filepaths.